src/elexmodel/utils/math_utils.py

```diff
@@ -30,7 +34,16 @@ def weighted_median(x, weights):
 
     # find index of largest x_i where weights are less than or equal 0.5
     weights_cumulative = np.cumsum(weights_sorted)
-    median_index = np.where(weights_cumulative <= 0.5)[0][-1]
+    # conformity scores are lined up in size order, but each is assigned a
+    # weight based on unit population. The list is split in half according to
+    # cumulative weights. But if the first element in the list is already over
+    # 50% of total weight, there will be nothing in one side of the list.In
+    # that case return the first element
+    if weights_cumulative[0] >= 0.5:
+        LOG.warning("Warning: smallest conformity value is greater than or equal to half the weight")
+        return x_sorted[0]
```

Comment:
is there a way to continue to use `median_index`? Like set `median_index` to zero? No, right? since in the most common case we would now return the second element from the left.